app/components/tables/holidays_table/HolidaysTable.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+    useReactTable,
+    flexRender,
+    getCoreRowModel,
+    getPaginationRowModel,
+    getFilteredRowModel,
+    getSortedRowModel,
+} from "@tanstack/react-table";
+import useAxiosPromise from "../../../hooks/useAxiosPromise";
```

Comment:
Let's try using TanStack Query instead of useAxiosPromise.ts. I think it would be a great dependency to use.

Reply:
Actually, let's make this a ticket.